Gleb_Yelizov/main.js

```diff
@@ -1,29 +1,139 @@
-document.addEventListener('DOMContentLoaded', render);
-
-const text = {
-    header: 'This is the file for your home tasks',
-    list: [
-        'Please, write your JavaScript code in <strong>main.js</strong>',
-        'Please, write your CSS code in <strong>styles.css</strong>',
-        'Please, write your HTML right here in <strong>index.html</strong>'
-    ]
+// document.addEventListener('DOMContentLoaded', render);
+
+// const text = {
+//     header: 'This is the file for your home tasks',
+//     list: [
+//         'Please, write your JavaScript code in <strong>main.js</strong>',
+//         'Please, write your CSS code in <strong>styles.css</strong>',
+//         'Please, write your HTML right here in <strong>index.html</strong>'
+//     ]
+// };
+
+// function render() {
+//     const main = document.createElement('main');
+//     const h2 = document.createElement('h2');
+//     h2.textContent = text.header;
+
+//     const ul = text.list.reduce((ul, text) => {
+//         const li = document.createElement('li');
+
+//         li.innerHTML = text;
+//         ul.append(li);
+
+//         return ul;
+//     }, document.createElement('ul'));
+
+//     main.append(h2);
+//     main.append(ul);
+//     document.body.append(main);
+// }
+
+//-------------------------------------------------------Homework Lesson_09-------------------------------------
+
+//-----------------------calculator------------------
+
+let calculator = {};
+
+function read() {
+    this.firstValue = +prompt("Введите первое число", '');
+    this.secondValue = +prompt("Введите второе число", '');
+}
+
+function sum() {
+   return this.firstValue + this.secondValue;
+}
+
+function mul() {
+    return this.firstValue * this.secondValue;
+}
+
+calculator.read = read;
+calculator.sum = sum;
+calculator.mul = mul;
```

Comment:
If I don't miss anything, according to the task (regarding the learned topic - functions-constructors and classes), you had to implement a function-constructor Calculator, which could to the same logic. it is important for understanding the topic.

```suggestion
function Calculator() {
    this.read = () => {
       this.firstValue = +prompt("Введите первое число", '');
       this.secondValue = +prompt("Введите второе число", '');
    };
    this.sum = () => {
       return this.firstValue + this.secondValue;
    };
    this.mul = () => {
       return this.firstValue * this.secondValue;
    };
}
```